internal/pkg/app/handlers_v1.go

```diff
@@ -58,23 +63,86 @@ type APIManager struct {
 	tenantStorer               tenant.TenantStorer
 	quotaManager               *quota.QuotaManager
 	jwtManager                 jwt.JWTConsumer
+	tenantCache                *TenantCache
 	addRouteSuccessRecorder    metric.BoundFloat64Counter
 	addRouteFailureRecorder    metric.BoundFloat64Counter
 	removeRouteSuccessRecorder metric.BoundFloat64Counter
 	removeRouteFailureRecorder metric.BoundFloat64Counter
+	globalWebhookOrg           string
+	globalWebhookApp           string
+	globalWebhookRouteId       string
+}
+
+type CachedTenantConfig struct {
```

Comment:
What do you think about using a 3rd party cache library: https://pkg.go.dev/github.com/ReneKroon/ttlcache/v2

Reply:
haha, you know how i feel about using other people's code...